samples/subsys/display/lvgl/boards/bl5340_dvk_cpuapp.conf

```diff
@@ -0,0 +1,3 @@
+CONFIG_KSCAN=y
+
+CONFIG_LV_Z_POINTER_KSCAN=y
```

Comment:
Please justify why this overlay is needed and can not be resolved at the board level.
You commit message is wrong.

Reply:
So I should add it to the Kconfig.defconfig?